.env.example

```diff
@@ -1,2 +1,4 @@
 DATABASE_URL=postgres://username:password@localhost:5432/db_name
 JWT_SECRET=secret
+SENDGRID_API_KEY=
+FRONTEND_URL=http://localhost:3000
```

Comment:
Can we also add an extra line after line 4

Reply:
@kimotho-njoki Yes, I have just added the extra line